README.md

```diff
@@ -49,14 +54,46 @@ Homebrew formula is here: [Cj-bc/homebrew-sonic-pi-tool](https://github.com/Cj-b
 
 ## Usage
 
+- [start-server](#start-server)
 - [check](#check)
 - [eval](#eval)
 - [eval-file](#eval-file)
 - [eval-stdin](#eval-stdin)
 - [stop](#stop)
 - [logs](#logs)
-- [start-server](#start-server)
 - [record](#record)
+- [sync](#sync)
+
+### `start-server`
+
+Attempts start the Sonic Pi server, if the executable can be found.
+
+Not supported on Windows.
+
+```sh
+sonic-pi-tool start-server
+# Sonic Pi server booting...
+# Using protocol: udp
+# Detecting port numbers...
+# ...
+```
+
+`start-server` will write a `config.toml` under `~/.sonic-pi-tool` with the appropriate information required to connect to the sonic pi audio server.
+
+### `sync`
+
+The Sonic Pi server allocates ports dynamically, and also establishes a token to communicate amongst the different processes involved.
+
+```sh
+sonic-pi-tool sync 33926 1839294199
+### Now sonic-pi-tool is sync'd with that server
```

Comment:
The user should be able to read this document and then know all _need_ to know to be able to use the tool, so they'll need to know what numbers to use here.